### PR TITLE
proof of concept for load data via Webpack

### PR DESCRIPTION
**Pull request by thangngoc89**
ref #11
There is no hot loading yet. Will figure it out

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -71,6 +71,7 @@
     "fs-extra": "^0.26.4",
     "js-beautify": "^1.5.10",
     "lnfs-cli": "^1.0.1",
+    "lodash": "^4.0.1",
```
Comment (MoOx) [inline package.json]: I really don't want to include lodash. We can do 90% of lodash with simple es6.

**Comment by MoOx** [comment]
I don't think it's possible to enhance the current codebase. This have to be rewritten. And the fact that we use dangerousinnerhtml make this "stupid" for hot loading.
Currently, json files are already emmited by webpack.

I don't really understand what this solution brings us.

The have hot loading we will have to use some `if (module.hot)` blocks to reload data when webpack emit a .hot.js file.

I think it will not be that easy.

File: src/redux/modules/index.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
 export collection from "./collection"
 export layouts from "./layouts"
 export pages from "./pages"
+export loadPage from "./loadPage"
```
Comment (MoOx) [inline src/redux/modules/index.js]: the loading to load page should be in "pages" module.

File: package.json (hunk 1 above)
Comment (thangngoc89) [inline package.json]: This is just a proof of concept when I finish. ok ?

File: src/redux/modules/index.js (hunk 2 above)
Comment (thangngoc89) [inline src/redux/modules/index.js]: loadPage should be in context like `layouts`

#97

File: package.json (hunk 1 above)
Comment (MoOx) [inline package.json]: ok :)

**Comment by thangngoc89** [comment]
Close as this is not the way to get this done :)

**Pull request closed without merging**